runner/src/server/plugins/engine/views/partials/heading.html

```diff
@@ -1,8 +1,8 @@
+{% if sectionTitle and (sectionTitle !== pageTitle) %}
+  <h2 class="govuk-caption-xl govuk-!-margin-top-0" id="section-title">{{sectionTitle}}</h2>
+{% endif %}
```

Comment:
Was this previously a bug? i.e. when `showTitle=false` we would also exclude the sectionTitle. Is there another place we were adding it, or was it just excluded entirely?